littlepay/commands/groups.py

```diff
@@ -69,6 +69,8 @@ def groups(args: Namespace = None) -> int:
                     print(f"{group.id},{product.id},{group.participant_id}")
                 else:
                     print(" ", product)
+        if command == "funding_sources":
```

Comment:
I'm going to nitpick again here, I think this should be an `elif` -- since we only want one of these things to happen 

* printing the products
* printing the funding sources
* printing the CSV output

This is minor, since this behavior is already represented by the logic (i.e. the command cannot be both `"products"` and `"funding_sources"` so only one of these `if` will ever hit...)

BUT I think from clarity of the code, it makes more sense as an `elif` so it is clearly part of the larger `if` block.

Reply:
Thanks for the clarification, that makes a lot of sense 👍